backend/src/Designer/Filters/IO/IoErrorCodes.cs

```diff
@@ -0,0 +1,8 @@
+namespace Altinn.Studio.Designer.Filters.IO;
+
+public class IoErrorCodes
+{
+    public const string DataModelCsharpNotFound = nameof(DataModelCsharpNotFound);
```

Comment:
Naming doesn't seem right. IO exceptions are common for all endpoints. Having "DataModel" in name doesn't seems right

Reply:
The idea here was to specify these at a later stage. Currently none of them are in use. But I can delete the file for now maybe?

Reply:
I think just renaming them to something else would be fine. For example "ResourceNotFound" and provide that error code in filter for error code instead of "File not found" text.